datahub-project/datahub
# Download Lineage Results Cypress Test

Test steps:

- Verify the downloaded CSV (urn column)
- Verify the total row count vs rows in CSV
- Test with 1st degree filter 2nd degree filter and 3rd degree filter
- Multi-Page Support

## Checklist
